ops/internal/config/chain.go

```diff
@@ -124,12 +123,13 @@ type GenesisRef struct {
 }
 
 type SystemConfig struct {
-	BatcherAddr       ChecksummedAddress `json:"batcherAddr" toml:"batcherAddress"`
-	Overhead          common.Hash        `json:"overhead" toml:"overhead"`
-	Scalar            common.Hash        `json:"scalar" toml:"scalar"`
-	GasLimit          uint64             `json:"gasLimit" toml:"gasLimit"`
-	BaseFeeScalar     *uint64            `json:"baseFeeScalar,omitempty" toml:"baseFeeScalar,omitempty"`
-	BlobBaseFeeScalar *uint64            `json:"blobBaseFeeScalar,omitempty" toml:"blobBaseFeeScalar,omitempty"`
+	BatcherAddr       ChecksummedAddress  `json:"batcherAddr" toml:"batcherAddress"`
+	BatchInboxAddr    *ChecksummedAddress `json:"batchInboxAddr" toml:"batch_inbox_addr"`
```

Comment:
This is an interesting one. The batch inbox address isn't actually in the [system config that gets derived](https://github.com/ethereum-optimism/optimism/blob/d032aea4ab029918b4ac0cec6f0a07de08590ed3/op-service/eth/types.go#L502-L502), just [the contract](https://github.com/ethereum-optimism/optimism/blob/fab8931ebed3c64ad3414b4badbdba8dce670146/packages/contracts-bedrock/src/L1/SystemConfig.sol#L282-L285).

The genesis system config vars, in my head, are things that can change (that the protocol supports changing in normal operation) - imo, it's fine to lift into `genesis`, but not `genesis.system_config`.

Reply:
Ahh ok got it. Will lift it one level to `genesis` then. And will update the op-geth and monorepo prs accordingly

Reply:
Maybe the inconsistency is that we would put this address that can be read from the system config (and that cannot be changed) into `genesis.SystemConfig`, but would still leave others in the `addresses` section. So @clabby @bitwiseguy wdyt about putting the batch inbox address into the `addresses` section instead?